<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>chat</class>
 <widget class="QWidget" name="chat">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>869</width>
    <height>718</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="sendButton">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>670</y>
     <width>121</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>send</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="msgLine">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>670</y>
     <width>321</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="chatLog">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>550</y>
     <width>321</width>
     <height>111</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_2">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>550</y>
     <width>121</width>
     <height>111</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="videoLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>600</width>
     <height>480</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
